Atomic/syscontainers.py

```diff
@@ -1157,38 +1153,8 @@ def _do_checkout(self, repo, options):
         if options["prefix"]:
             return options["values"]
 
-        sym = os.path.join(self._get_system_checkout_path(), options["name"])
-        if os.path.exists(sym):
-            os.unlink(sym)
-        os.symlink(options["destination"], sym)
-
         try:
```

Comment:
it looks like this chunk got lost in the new version.  I think we still need to create the symlink

Reply:
ah, this is done in the second commit. I mentioned the reason briefly in the commit msg. Going to mention it here again. The reason for the removal was mainly because of the duplicate calls. 

When prefix is not specified, we are creating a symlink already right? And then one line right below it, we directly return when there is prefix specified. So, the only case where we create a symlink is when prefix is None, which is covered in the following case right?

```
          if not options["prefix"]:
             sym = os.path.join(self._get_system_checkout_path(), options["name"])
             if os.path.exists(sym):
                 os.unlink(sym)
             os.symlink(options["destination"], sym)
 
         # if there is no container service, delete the checked out files.  At this point files copied to the host
         # are already handled.
         if not has_container_service:
             if not remote_path:
                 shutil.rmtree(os.path.join(options["destination"], "rootfs"))
             return options["values"]
 
         if options["prefix"]:
             return options["values"]
```

So, I believe the same creating symlink logic has been called twice originally? That's why I did the removal.

Reply:
True, we are already doing that.

We can drop this part as it is obviously not needed.